NOTICE

```diff
@@ -33,10 +34,10 @@ Unless required by applicable law or agreed to in writing, software distributed
 
 ==========================================
 
-Users who apply the Software resulting in presentations or papers are kindly asked to cite the following “Software Documentation Paper” alongside with the Software doi (doi:10.17874/ac8548f0315) and version number:
-Eyring et al., ESMValTool (v1.0) – a community diagnostic and performance metrics tool for routine evaluation of Earth System Models in CMIP, Geosci. Model Dev., 2016.
+Users who apply the Software resulting in presentations or papers are kindly asked to cite the following “Software Documentation Paper” alongside with the Software doi (doi:10.5281/zenodo.3952695) and version number:
+Righi et al., Earth System Model Evaluation Tool (ESMValTool) v2.0 - technical overview, Geosci. Model Dev., 2020.
 
```

Comment:
Did you only want to cite the Righi et al paper here? In the "Welcome letter" we have decided to list all four v2 papers.

Reply:
Since this is the NOTICE file for ESMValCore, I thought citing only Righi et al. would be in order. This is different in the NOTICE file for ESMValTool. There, all 4 papers are cited? Does that sound OK? I am also fine putting all four papers here.

Reply:
Yep, I saw the four papers listed for the ESMValTool. I think you are right, the Righi et al. paper should be enough here since this is the NOTICE about the ESMValCore.